tracks/csharp/exercises/hamming/mentoring.md

```diff
@@ -82,6 +82,13 @@ public static int Distance(string strand1, string strand2)
 }
 ```
 
+### A single line method using lambda operator
```

Comment:
I wouldn't necessarily want to encourage using a single line. Maybe rephrase this as just:

```suggestion
### A single expression
```